Translation/Tests/TranslationTests.swift

```diff
@@ -0,0 +1,40 @@
+import XCTest
+@testable import Translation
+import Core
+import AsyncHTTPClient
+import NIO
+
+class TranslationTests: XCTestCase {
+    var elg: MultiThreadedEventLoopGroup!
+    var client: HTTPClient!
+
+    override func setUp() {
+        super.setUp()
+        elg = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+        client = HTTPClient(eventLoopGroupProvider: .shared(elg),
+                            configuration: .init(ignoreUncleanSSLShutdown: true))
+    }
+
+    //Just for manual testing purposes.
```

Comment:
@pballart Looks good to me!. Just confirming that you went ahead and tested this against the actual API and this test isn't needed/won't run.

Reply:
Correct. I used a personal credentials file injected via env vars and tested the real API. As per if it will run I am not sure so let me comment it just in case.